.env.example

```
@@ -0,0 +1,2 @@
+GATSBY_GTM_ID=GTM-XXXXXXX
+GATSBY_GTM_SUPPORT=false
```

Non-blocking:

Hmmm, I think Google tag manager is not really realted to gatsby so I'm not sure if those names won't confuse anyone in the future. Maybe we could change them to just:

```suggestion
GOOGLE_TAG_MANGAER_ID=GTM-XXXXXXX
GOOGLE_TAG_MANAGER_SUPPORT=false
```

?

---

To access environment variables in a client they need to be prefixed with `GATSBY_` so Gatsby knows to expose them on build.

https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/#accessing-environment-variables-in-the-browser

---

Good catch 👍